src/database/remove.js

```diff
@@ -0,0 +1,189 @@
+"use strict";
+
+var api = require("../api");
+var FirebaseError = require("../error");
+var request = require("request");
+var responseToError = require("../responseToError");
+var pathLib = require("path");
+var utils = require("../utils");
+var Queue = require("../queue");
+const logger = require("../logger");
+
+/**
+ * Construct a new RTDB delete operation.
+ *
+ * @constructor
+ * @param {string} instance the RTDB instance ID.
+ * @param {string} path path to delete.
+ * @param {boolean} options.concurrency the number of concurrent chunk delete allowed
+ * @param {boolean} options.retires the number of retries for each chunk delete
+ */
+function DatabaseRemove(instance, path, options) {
```

Comment:
I know you were probably following conventions elsewhere in the CLI, but we are now on Node v6+, which means we can actually use ES6 classes. So let's refactor this into a full-on `class`.